analyzedphenotypes.module

```diff
@@ -311,6 +308,20 @@ function analyzedphenotypes_data_json($page_argument_5, $page_argument_6, $key =
 
       break;
 
+    //
```

Comment:
Make sure that each case in this function has documentation of:
- where it is used
- what it is for
- what you expect the options to look like

Additionally, the design of this function makes me nervous. In my experience it is too easy to forget a `break;` and end up with additional options which might be hard to debug. Perhaps, at a minimum you should add a `$data_JSON = array();` at the beginning of each case to ensure there is no flow over if a break is missed. Personally, I would have made each function pulling options (e.g. analyzedphenotypes_cvprop) simply output the JSON as well.

Reply:
Upon further digging, I noticed this design of using switch statements to combine multiple, loosely related pieces of functionality into a single function is prevalent. Thus for this PR, keep the design with the switch and simply add the `$data_JSON = array();` at the beginning of each case.

Reply:
Updated doc block and removed unused case.